css/members.css

```diff
@@ -161,15 +157,3 @@ but underneath basic-details. Thus, the inner-details div will be visible but it
     font-size: 26px;
     color: white;
 }
-
-/* Animation for inner-details div, activated on hover */
-@keyframes fadeInUp {
-    from {
-        opacity: 0;
-        transform: translateY(10px);
-    }
-    to {
-        opacity: 1;
-        transform: translateY(0px);
-    }
-}
```

Comment:
Undo **ALL** fadeIn and fadeOut changes you made. The old version seemed more responsive. The simplifications you made to `members.css` is okay though

Reply:
Done